src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -130,7 +130,7 @@ def acr_update_get(client,
     )
 
 def acr_update_custom(instance,
-                      admin_enabled=None,
+                      admin_enabled,
```

Comment:
Everywhere else in the CLI, the create exposes a flag and the update a binary choice list (either true/false or enabled/disabled).  So I don't think it would be confusing. The primary things on updates is that you leave everything as is except what was specifically specified.